src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class DeleteCommand extends Command {
+    public DeleteCommand(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Delete the task from list at a specific index if index is valid, save the task list and display to user.
+     *
+     * @param  tasks   the task list
+     * @param   storage the storage object to save the list
+     * @param   ui  the ui object to interact with user
```

Comment:
The white space between "@param" and the parameter name should be standardized to 1 white space.